03.1/middlwares/error.middlware.ts

```diff
@@ -22,6 +24,15 @@ const validationErrorMiddleware = (
 };
 
 const errorMiddleware = (err: any, req: Request, res: Response, next: NextFunction): void => {
+  const { method, query, body, originalUrl } = req;
+  const queryStr = JSON.stringify(query);
+  const bodyStr = JSON.stringify(body);
```

Comment:
Could you please avoid saving password to logs?

Reply:
Thanks, updated.